tests/lru_cache_test.py

```diff
@@ -150,6 +151,35 @@ def test_max_size(self):
     self.assertIsNone(cache.get("cache-a"))
     self.assertEqual(set(self.path.glob("cache-*")), set())
 
+  def test_multiprocess(self):
+    large_number = 1024
+
+    bytes_a = b"a" * (large_number + 1)
+    bytes_b = b"a" * (large_number + 1)
+
+    cache = LRUCache(self.name, max_size=large_number * 2)
```

Comment:
Maybe make this local to `call_multiple_puts_and_gets`, so that each process has its own instance?

And then create one local to the test process on L180.